streaming/java/streaming-state/src/test/java/org/ray/streaming/state/keystate/state/impl/OperatorStateImplTest.java

```diff
@@ -0,0 +1,99 @@
+package org.ray.streaming.state.keystate.state.impl;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.ray.streaming.state.backend.OperatorStateBackend;
+import org.ray.streaming.state.backend.impl.MemoryStateBackend;
+import org.ray.streaming.state.keystate.desc.ListStateDescriptor;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+
+public class OperatorStateImplTest {
+
+  OperatorStateImpl<Integer> operatorState;
+  ListStateDescriptor<Integer> descriptor;
+  OperatorStateBackend operatorStateBackend;
+
+  @BeforeClass
+  public void setUp() throws Exception {
+    String table_name = "kepler_hbase_operator";
+    Map config = new HashMap<>();
+
+    operatorStateBackend = new OperatorStateBackend(new MemoryStateBackend(config));
+
+    descriptor = ListStateDescriptor
+      .build("OperatorStateImplTest" + System.currentTimeMillis(), Integer.class, true);
+    descriptor.setNumber(1);
+    descriptor.setIndex(0);
+    descriptor.setTableName(table_name);
+
+    operatorState = (OperatorStateImpl<Integer>) operatorStateBackend.getSplitListState(descriptor);
+  }
+
+  //@Test
```

Comment:
Use `@Ignore` if there is some temporary issue, or remove the case if we don't need it any more.

Reply:
ok